File: javascript/Fibonacci.js

function gerarFibonacci() {

    let ultimoTermo = parseInt(prompt("Digite o último termo da sequência: "))

    const fibonacci = [0, 1];

    for (let i = 2; i < ultimoTermo; i++) {
        const sucessor = fibonacci[i - 1] + fibonacci[i - 2];
        fibonacci.push(sucessor);
    }
    return fibonacci;
}

// Chamada da função

gerarFibonacci();

File: java/Fibonacci.java
package java;

public class Fibonacci {
  public void gerarFibonacci(int n) {
    int a = 0, b = 1;
    System.out.println("Sequência de Fibonacci até o termo " + n + ":");
    for (int i = 0; i < n; i++) {
      System.out.print(a + " ");
      int c = a + b;
      a = b;
      b = c;
    }
    System.out.println();
  }
}